MatrixSDK/MXSession.h

```diff
@@ -59,6 +59,11 @@ typedef enum : NSUInteger
      Data from the MXStore has been loaded.
      */
     MXSessionStateStoreDataReady,
+    
+    /**
+     Data from local cache is being processed.
```

Comment:
Local cache is a bit confusing. We usually use it for MXStore.
Let me try this:

```suggestion
     Background sync cache is being processed. More local data will come.
```

Reply:
Updated.